public/js/main-dev.js

```diff
@@ -172,7 +232,7 @@
 
         // draw gravity lines
         this.ctx.beginPath();
-        if (this.isNightMode()) {
```

Comment:
This is a DOM read – a bit slow to do 60 times per second, that's why I changed it back..